src/Paket.Core/PackageResolver.fs

```diff
@@ -651,7 +651,7 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
 
     let rec step (stage:Stage) (stackpack:StackPack) compatibleVersions (flags:StepFlags) =
 
-        let fuseConflicts currentConflict priorConflictSteps =
+        let inline fuseConflicts currentConflict priorConflictSteps =
```

Comment:
Ah okey, I thought the thing is now tail recursive but the compiler is not picking it up because of this in debug mode. Thanks for the clarification.